tasks/dataverse-previewers.yml

```diff
@@ -4,12 +4,33 @@
   debug:
     msg: '##### DATAVERSE PREVIEWERS #####'
 
+- name: download and unzip dataverse previewers to apache server
+  unarchive:
+    src: '{{ dataverse.previewers.zip_url }}'
+    dest: '{{ dataverse.previewers.dir }}'
+    remote_src: yes
+    owner: root
+    group: root
+    mode: "u+x,g+x,o+x"
```

Comment:
@mderuijter : probably not necessary to make these executable.

Reply:
Tested it, and apparently it is required.